src/components/nc-spinner.vue

```diff
@@ -0,0 +1,162 @@
+<template>
+  <div v-show="active" :class="{ veil: hasVeil }" class="nc-spinner">
+    <div class="nc-spinner__container">
+      <div class="nc-spinner__icon">
```

Comment:
Siguiendo BEM estrictamente este elemento ya no debería tener el prefijo nc-spinner (nc-progress) por su posición.
```suggestion
      <div class="icon">
```